app/models/manageiq/providers/nuage/network_manager/refresher.rb

```diff
@@ -0,0 +1,2 @@
+class ManageIQ::Providers::Nuage::NetworkManager::Refresher
```

Comment:
Should subclass [BaseManager::Refresher](https://github.com/ManageIQ/manageiq/blob/2ee878bacafaa9e444c9a50628aa5d653b4a197d/app/models/manageiq/providers/base_manager/refresher.rb).